sn_interface/src/messaging/authority.rs

```diff
@@ -62,7 +62,7 @@ impl NodeAuth {
 
 /// Authority of a single peer that uses it's BLS Keyshare to sign the message.
 #[derive(Clone, Debug, Eq, PartialEq, serde::Deserialize, serde::Serialize)]
-pub struct BlsShareAuth {
+pub struct SectionAuthPart {
```

Comment:
We've been using to the `Share` terminology throughout other modules. I think it's descriptive enough and it's used in the papers as well (i.e. threshold crypto work always refers to "shares").

```suggestion
pub struct SectionAuthShare {
```